### PR TITLE
data: poojasanwal

### PR DESCRIPTION
**Pull request by poojasanwal**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5211"></a>

### CODE REVIEW

File: poojasanwal.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Pooja Sanwal",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "She/Her, I talk about development, technical writing, open source and communities",
+  "avatar": "https://github.com/poojasanwal.png",
```
Comment (SurajPratap10) [inline poojasanwal.json:6]:
```suggestion
  "type": "personal",
  "bio": "She/Her, I talk about development, technical writing, open source and communities",
```

Comment (SurajPratap10) [reply poojasanwal.json:6]: We don't need the `displayStatsPublic` and `avatar` after the recent update, So remove these 2 lines.

**Review by priyankarpal — CHANGES_REQUESTED** [review]
Move `poojasanwal.json` into `data/poojasanwal.json`

Screenshot from 2023-03-02 23-52-01

File: poojasanwal.json (hunk 1 above)
Comment (poojasanwal) [reply poojasanwal.json]: Thanks for reviewing. I have removed the two lines.

**Review by SurajPratap10 — CHANGES_REQUESTED** [review]
Move `poojasanwal.json` under `data` folder

File: poojasanwal.json (hunk 1 above)
Comment (SurajPratap10) [reply poojasanwal.json]: > Thanks for reviewing. I have removed the two lines.

ya, but move it under `data` folder

Comment (SurajPratap10) [reply poojasanwal.json]: > > Thanks for reviewing. I have removed the two lines.
> 
> ya, but also move it under the `data` folder

**Review by SurajPratap10 — APPROVED** [review]
LGTM

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**